### DISCUSSION TITLE
Default policy for `reexecuteQuery`

### DISCUSSION THREAD

**Discussion by pogo5422** [urql-graphql/urql#2140] [Q&A] [answered]
Is there a way to separately set a policy for `reexecuteQuery` and usual request from hook? E.g. I want to have `cache-first` policy behavior when my component just updates, but calling `reexecuteQuery` is always should be made as `network-first` and I'd prefer to omit specifying options in every single `reexecuteQuery` call.

```typescript
export const client = createClient({
  requestPolicy: "cache-first",
  reexecuteRequestPolicy: "network-first",
})
```

**Comment by JoviDeCroock** [comment, marked as answer]
This is a good idea, we might want to convert this into an RFC and move it onto a candidate.
